public/index.html

```diff
@@ -6,8 +6,169 @@
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <title>Untitled</title>
         <link rel="author" href="humans.txt">
+        <link> <link rel="stylesheet" type="text/css" href="facesmashmain.css">
+    
+    <title>"Facesmash"</title>          
+    
     </head>
+    
     <body>
-        <h1>Audacious Aretes</h1>
+        <ul class="fixedheader clearfix">
+            <li><h1><a href = "/">Facesmash</a></h1></li>
+                <li>
```

Comment:
this is just a minor thing, make sure it aligns with the indentation for it's siblings.